lib/tools/getSpotifyArtistTopTracks.ts

```
@@ -0,0 +1,67 @@
+import { z } from "zod";
+import { tool } from "ai";
+
+/**
+ * Response type for Spotify Get Artist Top Tracks
+ */
+export interface GetSpotifyArtistTopTracksResponse {
+  tracks: Array<{
+    album: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)